<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoLoadErrorDialog</class>
 <widget class="QDialog" name="PhotoLoadErrorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>No Valid geotag</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="photo">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>This photo do not contain geo-tagging EXIF data.
What do you want to do next?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbMatch">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Try to match with a track node</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbLoad">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Load without associated node</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbBarcode">
     <property name="text">
      <string>Extract info from barcode (Walking Papers)</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbIgnore">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Ignore it</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbRemember">
     <property name="text">
      <string>Do this for all current photos.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbMatch</sender>
   <signal>clicked()</signal>
   <receiver>PhotoLoadErrorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbLoad</sender>
   <signal>clicked()</signal>
   <receiver>PhotoLoadErrorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbIgnore</sender>
   <signal>clicked()</signal>
   <receiver>PhotoLoadErrorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbBarcode</sender>
   <signal>clicked()</signal>
   <receiver>PhotoLoadErrorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
